src/emmetHelper.ts

```diff
@@ -169,6 +171,10 @@ function getCurrentWord(document: TextDocument, position: Position): string {
 	}
 }
 
+function makeCursorsGorgeous(expandedWord: string): string {
+	return expandedWord.replace(/\$\{\d+\}/g, '|').replace(/\$\{\d+:([^\}]+)\}/g, '_$1_');
```

Comment:
I am not sure about the surrounding the placeholders with `_`.
Will remove that for now and wait for user feedback